bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -95,7 +95,6 @@ <h1 class="large" id="main-headline">
       {% if l10n_has_tag('firefox_ios') %}
       <li class="hide-for-refresh"><a href="{{ firefox_ios_url('mozorg-uptodate-appstore-link') }}" data-link-type="download" data-download-os="iOS">{{ _('Add Firefox to your iPhone, iPad and iPod touch') }}</a></li>
       {% endif %}
-      <li class="hide-for-refresh"><a href="{{ url('firefox.os.index') }}">{{_('Learn about Firefox OS')}}</a></li>
```

Comment:
That line should only appear to people using a Firefox OS device, so I decided to leave it. Of course, we could still remove it and I bet it would do no harm... I'll confirm what we should do there.